dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1412,7 +1410,7 @@ def refute_bad_gateway_or_site_unreachable
     And I save the section id from row 0 of the section table
     Given I create a student named "#{student_name}"
     And I join the section
-  }
+  GHERKIN
```

Comment:
What do you think about using `STEPS` instead of `GHERKIN`? I don't think the average UI test writer needs to know that this syntax is called gherkin (I certainly didn't until just now!)

Reply:
I very much prefer we use the heredoc name to specify the syntax being used, as the Ruby style guide generally recommends. Particularly in this case where there's already a variable name to specify what this content is going to be used for; using `STEPS` here doesn't add any helpful information to the dev reading this code, but by explicitly using `GHERKIN`we can make it easier for devs to find relevant information.

Reply:
fair enough!